test/FunctionalTests/Microsoft.OData.Edm.Tests/Csdl/CsdlReaderTests.cs

```diff
@@ -1077,5 +1077,24 @@ public void ParsingPropertyWithCoreTypeDefinitionAsPropertyTypeWorks()
             Assert.Equal(EdmTypeKind.TypeDefinition, property.Type.TypeKind());
             Assert.Equal("Org.OData.Core.V1.LocalDateTime", property.Type.FullName());
         }
+
+        [Theory]
+        [InlineData("4.0")]
+        [InlineData("4.01")]
+        public void ValidateEdmxVersions(string odataVersion)
+        {
+            string xml = "<?xml version=\"1.0\" encoding=\"utf-16\"?><edmx:Edmx Version=\"" + odataVersion + "\" xmlns:edmx=\"http://docs.oasis-open.org/odata/ns/edmx\"><edmx:DataServices /></edmx:Edmx>";

```

Comment:
>edmx:Edmx Version=\"" + odataVersion [](start = 70, length = 36)

does it mean "Edmx version is same as Edm version"?

Reply:
I have no idea why the current library differentiates between EdmVersion and EdmxVersion; they should always be identical. Since these are public I didn't change the redundancy, but in 8.x we should eliminate the redundancy and have a cleaner model for a single version.

---
In reply to: [290417063](https://github.com/OData/odata.net/pull/1476#discussion_r290417063) [](ancestors = 290417063)